std/functional.d

```diff
@@ -11,38 +11,38 @@ $(SCRIPT inhibitQuickIndex = 1;)
 $(BOOKTABLE ,
 $(TR $(TH Function Name) $(TH Description)
 )
-    $(TR $(TD $(D $(LREF adjoin)))
+    $(TR $(TD $(LREF adjoin))
         $(TD Joins a couple of functions into one that executes the original
         functions independently and returns a tuple with all the results.
     ))
-    $(TR $(TD $(D $(LREF compose)), $(D $(LREF pipe)))
+    $(TR $(TD $(LREF compose)), $(LREF pipe)
```

Comment:
It should be `$(TR $(TD $(LREF compose), $(LREF pipe))`. The regular expression didn't handle nested parenthesis correctly and now the second link will show up outside the table cell.